benefits.md

```diff
@@ -122,15 +122,15 @@ There is no sense in working through a cold. Stay home, take the day off, and fo
 
 ## Family Leave
 
-> We need to discuss this as a team. It would be great to provide a sizeable leave (paternity/maternity/family health issues), but we need to figure out how to reduce the financial impact ([issue #1](https://github.com/b2io/handbook/issues/1))
+> FMLA policy is currently under review [here](https://docs.google.com/document/d/17D9HrzaxjHW7keer-3s4IibPaxcQylMpBve549kNW6Q/edit?usp=sharing)
```

Comment:
Should we just move the FMLA policy to the handbook as a pr to close that referenced issue? (and hold off on these changes until it's merged?)

Reply:
Just reviewed that document and it seems pretty unfinished at this point.  There's still a lot of unclear language there that given my upcoming need to use it I'd like to clarify before we bake it into the handbook.

Reply:
Yeah, I guess my suggestion would be to link that wip document on the issue instead of including it in the handbook at this point.

Reply:
Ahh totally makes sense, misread  :ignore me: